test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -566,9 +566,75 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 			By("Saving the old performance profile")
 			initialProfile = profile.DeepCopy()
 		})
+		When("RealTime Workload with RealTime Kernel set to false", func() {
+			It("[test_id:50991][crit:high][vendor:cnf-qe@redhat.com][level:acceptance]should update kernel arguments and tuned accordingly", func() {
```

Comment:
test_id shouldn't shown, right?

Reply:
we add test case id to all the test cases in functional tests. As that's the only way to map the test cases in automation to test cases in our test case management system